shell/platform/common/flutter_platform_node_delegate.h

```diff
@@ -129,11 +129,17 @@ class FlutterPlatformNodeDelegate : public ui::AXPlatformNodeDelegateBase {
   ///             Subclasses must call super.
   virtual void Init(std::weak_ptr<OwnerBridge> bridge, ui::AXNode* node);
 
- protected:
   //------------------------------------------------------------------------------
-  /// @brief      Gets the underlying ax node for this accessibility node.
+  /// @brief      Gets the underlying ax node for this platform node delegate.
   ui::AXNode* GetAXNode() const;
 
+  //------------------------------------------------------------------------------
+  /// @brief      Gets the owner of this platform node delegate. This is useful
+  ///             when you want to get the information surround this platform
+  ///             node delegate, e.g. the global rect of this platform node
+  ///             delegate.
```

Comment:
Please mention which task runner this is safe to use on. I assume Platform.

Reply:
I am not too familiar with this part, what makes some thing safe to run on platform thread vs the other. Also The accessibility birdge class should only be used in embedding code, can embedding code run things on UI thread?

Reply:
Fml weaks are only safe to use on the thread they were created on.  The embedder can access the other task runners.